guides/release/templates/handlebars-basics.md

```diff
@@ -204,11 +204,11 @@ The user will see a value of `3` rendered in the template!
 
 Ember ships with several built-in helpers, which you will learn more about in the following guides.
 
-### Nested Helpers
+### Nested Helper Invocations
```

Comment:
Should this be `Statements` instead of `Invocations`? 

I'm worried about `invocation` sounding a bit scary to newcomers.

Reply:
```suggestion
### Nested Helpers
```

I would like to keep this as "nested helpers" since it is a pre-existing anchor tag. Whenever possible, we try to keep our URLs alive, including anchors. Exceptions are when the topic naming is outdated.

Reply:
Seems good, my concern is _mostly_ that folks will think of "nested helpers" in the same way that they do "nested components" (which usually means "components in subdirectories").